.github/workflows/ci.yml

```diff
@@ -2,41 +2,60 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    steps:
```

Comment:
`build`/`test`/`lint` are unchanged except split into multiple jobs with `needs: build` added. looks like this might have broke CI